docs/DeveloperGuide.md

```diff
@@ -548,11 +551,11 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`   | CS2100 TA                                   | use tags to identify which labs are marked or unmarked                                           | know which what labs to mark next.
 | `*`     | CS2100 TA with multiple classes             | filter the contact list by name, classes, email  | easily identify those in the current class.                |                                                                        
 | `*`     | archive previous batch statistics           | compare current batch performance with them                                                      |
```

Comment:
Do you want to correct this now too? #473

Reply:
done!